src/software/ai/intent/movespin_intent.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "software/ai/intent/intent.h"
+#include "software/primitive/movespin_primitive.h"
+
+class MoveSpinIntent : public Intent
```

Comment:
What's the difference between this and a `SpinningMoveIntent`?

Reply:
whoops bad merge, movespin should not exist